submissions/UA-girl/task_js_dom/js/script.js

```diff
@@ -0,0 +1,199 @@
+const IMG_CLASS_NAME = 'card-image';
+const CARD_DESCRIPTION_CLASS_NAME = 'card-description';
+const CARD_TITLE_CLASS_NAME = 'card-header';
+const CARD_TEXT_CLASS_NAME = 'card-text';
+const CARD_LINK_CLASS_NAME = 'link-wiki';
+const MENU_LIST_NAME = 'catalog-list';
+const MENU_ITEM_NAME = 'catalog-item';
+const MENU_LINK_NAME = 'catalog-link';
+const KEY_CODE_SPACE = 32;
+const catalogItemsList = [
+    {
+        id: 0,
+        title: 'Siamese cat',
+        text: 'The Siamese cat is one of the first distinctly recognized breeds of Asian cat. Derived from the ' +
```

Comment:
It's not a blocker, but you can just use template strings or leave long strings. No need to concatenate them.

Reply:
I've made it on purpose to make code look nicer - to long strings are ugly and my editor does not automatically wrap strings for me.

Reply:
Got it. You can use formatting file settings in your IDE. For example, in VS code you can configure your files in order the content contains only screen width.